src/Examples/UnitModels/Operations/statejunction_testing.ipynb

```diff
@@ -0,0 +1,485 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: StateJunction Unit Model with Modular Property Package"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.core.display import SVG\n",
```

Comment:
I've tried doing this before and it works locally, but it causes errors on GitHub. If I were to guess what the issue is, it would be because the images I created do not have a transparent background (I've tried doing this myself but couldn't find a way that didn't involve manually doing the tedious work of scrubbing out the background). All of the image files from idaes/models/unit_models/icons are transparent and work fine when printing the image as part of the Markdown.